content/kubeone/main/guides/cis-benchmarking/cis1.8-kubeone1.7-k8s1.27/_index.en.md

```diff
@@ -0,0 +1,574 @@
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.15: Ensure that the scheduler.conf file permissions are set to 600 or more restrictive (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.16: Ensure that the scheduler.conf file ownership is set to root:root (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.17: Ensure that the controller-manager.conf file permissions are set to 600 or more restrictive (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.18: Ensure that the controller-manager.conf file ownership is set to root:root (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.19: Ensure that the Kubernetes PKI directory and file ownership is set to root:root (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.20: Ensure that the Kubernetes PKI certificate file permissions are set to 600 or more restrictive (Manual)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.21: Ensure that the Kubernetes PKI key file permissions are set to 600 (Manual)
+
+**Result:** 🟢 Pass
+
+---
+### 1.2. API Server
+#### 1.2.1: Ensure that the --anonymous-auth argument is set to false (Manual)
+
+**Result:** 🔴 Fail
+
+_The issue is under investigation to provide a fix in a future KubeOne release_
+
+---
+#### 1.2.2: Ensure that the --token-auth-file parameter is not set (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.3: Ensure that the --DenyServiceExternalIPs is set (Manual)
+
+**Result:** 🔴 Fail
+
+_The issue is under investigation to provide a fix in a future KubeOne release_
+
+---
+#### 1.2.4: Ensure that the --kubelet-client-certificate and --kubelet-client-key arguments are set as appropriate (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.5: Ensure that the --kubelet-certificate-authority argument is set as appropriate (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.6: Ensure that the --authorization-mode argument is not set to AlwaysAllow (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.7: Ensure that the --authorization-mode argument includes Node (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.8: Ensure that the --authorization-mode argument includes RBAC (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.9: Ensure that the admission control plugin EventRateLimit is set (Manual)
+
+**Result:** 🟢 Pass
+
+**Details:** EventRateLimit admission control plugin in in Alpha state, please see [here][eventratelimit]
+
+---
+#### 1.2.10: Ensure that the admission control plugin AlwaysAdmit is not set (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.11: Ensure that the admission control plugin AlwaysPullImages is set (Manual)
+
+**Result:** 🔴 Fail
+
+_The issue is under investigation to provide a fix in a future KubeOne release_
+
+---
+#### 1.2.12: Ensure that the admission control plugin SecurityContextDeny is set if PodSecurityPolicy is not used (Manual)
+
+**Result:** 🟢 Pass
+
+**Details:** SecurityContextDeny admission control plugin is deprecated as of [Kubernetes 1.27][securitycontextdeny]
```

Comment:
When I think better, this is also a fail, for the same reason as the previous comment. We don't enable this admission control plugin and that's all what CIS cares about. We can say that this audit failed but that we don't support enabling this admission control plugin because the official Kubernetes docs state that it shouldn't be used at all and that it's deprecated.